### ISSUE TITLE
Can't do nested 'where' or 'orderBy'

### ISSUE THREAD

**Issue by sergey94** [prisma/prisma#4062]
I can't submit a query which looks like following:
```
query {
  users {
    id
    fName
    sName
    pets(where: {
      type: "dog"
    }) {
      id
      name
    }
  }
}
```
or
```
query {
  users {
    id
    fName
    sName
    pets(orderBy: {
      name: asc
    }) {
      id
      name
    }
  }
}
```

Here is some parts of my source code.
```
...
const Query = queryType({
  definition(t) {
    t.crud.user();
    t.crud.users({
      filtering: true,
      ordering: true,
    });
    t.crud.pet();
    t.crud.pets({
      filtering: true,
      ordering: true,
    });
  },
});
...
const Pet = objectType({
  name: 'Pet',
  definition(t) {
    for(let fieldName in t.model) {
      t.model[fieldName]();
    }
  },
});
...
const User = objectType({
  name: 'User',
  definition(t) {
    for(let fieldName in t.model) {
      t.model[fieldName]();
    }
  },
});
...
```

I'm using Prisma 2/Nexus/Apollo.

## Environment
Mac OS
PostgreSQL
Prisma 2.9.0
Node 12.14.1

**Comment by pantharshit00** [comment]
Hey @sergey94 

Can you please provide your schema so that I can try reproducing this? Ideally please make a git repo with a minimal reproduction so that it is accurate and easier for me to identify the problem if there is any.

**Comment by sergey94** [comment]
Hey,
I guess the problem is anyway connected with nexusjs but here is some parts from my source code:

From *schema.prisma* (I have dropped unnecessary fields).
```
model User {
  id                          String               @id @default(cuid())
  fName                       String?
  sName                       String?
  pets                        Pet[]                     @relation("UserPet")
}

model Pet {
  id                 String               @id @default(cuid())
  name               String?
  user               User                 @relation(name: "UserPet", fields: [userId], references: [id])
  userId             String
}
```

And this is my GraphQL initial file:
```
const { PrismaClient } = require('@prisma/client');
const { nexusSchemaPrisma } = require('nexus-plugin-prisma/schema');
const { makeSchema, queryType, mutationType } = require('@nexus/schema');
const path = require('path');
const { ApolloServer } = require('apollo-server-koa');
const resolvers = require('./../resolvers');

const prisma = new PrismaClient();

const Query = queryType({
  definition(t) {
    t.crud.user();
    t.crud.users({
      filtering: true,
      ordering: true,
    });
    t.crud.pet();
    t.crud.pets({
      filtering: true,
      ordering: true,
    });
  },
});

const schema = makeSchema({
  types: [...resolvers, Query, Mutation],
  plugins: [nexusSchemaPrisma({
    experimentalCRUD: true,
  })],
  outputs: {
    schema: path.join(__dirname, './../../generated/schema.graphql'),
    typegen: path.join(__dirname, './../../generated/nexus.ts'),
  },
  nonNullDefaults: {
    input: false,
    output: false,
  },
});

const server = new ApolloServer({
  schema,
  playground: true,
  context: async (request) => ({
    request,
    prisma,
  }),
});

module.exports = server;
```

In resolvers I have:
```
const User = objectType({
  name: 'User',
  definition(t) {
    for(let fieldName in t.model) {
      t.model[fieldName]();
    }
  },
});
...
const Pet = objectType({
  name: 'Pet',
  definition(t) {
    for(let fieldName in t.model) {
      t.model[fieldName]();
    }
  },
});
```

It generates *schema.graphql*, whare the *User* is like this:
```
type User {
  fName: String
  id: String!
  pets(after: PetWhereUniqueInput, before: PetWhereUniqueInput, first: Int, last: Int): [Pet]
  sName: String
}
```
although it should be like:
```
type User {
  fName: String
  id: String!
  pets(after: PetWhereUniqueInput, before: PetWhereUniqueInput, first: Int, last: Int, orderBy: PetOrderByInput, where: PetWhereInput): [Pet]
  sName: String
}
```

So I manually do the following and it helps, but in previously it was automatic if I am not mistaken.
```
const petsFieldExtension = extendType({
  type: 'User',
  definition: (t) => {
    t.list.field('pets', {
      type: 'Pet',
      args: {
        where: 'PetWhereInput',
        orderBy: 'PetOrderByInput',
        after: 'PetWhereUniqueInput',
        before: 'PetWhereUniqueInput',
        first: 'Int',
        last: 'Int',
      },
      nullable: true,
      resolve: async (root, args, ctx) => {
        const pets = await ctx.prisma.pet.findMany(args);
        return pets;
      }
    });
  }
});
```

**Comment by pantharshit00** [comment]
Hey @sergey94 

This issue belongs to https://github.com/graphql-nexus/schema. I can't transfer it cross-org so please open a new issue there. Thanks!
